Add network tags support for Google Batch (#5951)
nextflow-io/nextflow: docs/reference/config.md

  - projects/{project}/global/networks/{network}
  - global/networks/{network}

`google.batch.networkTags`

> : Define the network tags to be applied to the instances created by Google Batch jobs. Network tags are used to apply firewall rules and control network access (e.g. `['allow-ssh', 'allow-http']`). 

Reviewer: ```suggestion
: The network tags to be applied to the instances created by Google Batch jobs. Network tags are used to apply firewall rules and control network access (e.g., `['allow-ssh', 'allow-http']`). 
```

> : Note network tags are ignored when using instance templates. See the [Google Cloud documentation](https://cloud.google.com/vpc/docs/add-remove-network-tags).

Reviewer: ```suggestion
: Network tags are ignored when using instance templates. See [Add network tags](https://cloud.google.com/vpc/docs/add-remove-network-tags) for more information.
```
